src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -0,0 +1,568 @@
+            }
+
+            yield return new object[]
+            {
+                "cookie: name=value;$port=\"80\";$Path=path;$Domain=domain", new CookieCollection()
+                {
+                    new Cookie("name", "value") { Port = "\"80\"", Path = "path", Domain = "domain" }
+                }
+            };
+
+            yield return new object[] { "cookie: ", new CookieCollection() };
+            yield return new object[] { "Unknown-Header: Test", new CookieCollection() };
+        }
+
+        [Theory]//[ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        [MemberData(nameof(Cookies_TestData))]
+        public async Task Cookies_Get_ReturnsExpected(string cookieString, CookieCollection expected)
+        {
+            await GetRequest("POST", null, new[] { cookieString }, (_, request) =>
+            {
+                Assert.Equal(expected.Count, request.Cookies.Count);
+                for (int i = 0; i < expected.Count; i++)
+                {
+                    Assert.Equal(expected[i].Name, request.Cookies[i].Name);
+                    Assert.Equal(expected[i].Value, request.Cookies[i].Value);
+                    // Cookie parsing is busted with .NET Core. Port, Path and Domain are all
+                    // set differently.
+                    if (PlatformDetection.IsFullFramework)
+                    {
+                        Assert.Equal(expected[i].Port, request.Cookies[i].Port);
+                        Assert.Equal(expected[i].Path, request.Cookies[i].Path);
+                        Assert.Equal(expected[i].Domain, request.Cookies[i].Domain);
+                    }
+                }
+            });
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        [InlineData("cookie: =value", false)]
+        [InlineData("cookie: $Version", true)]
+        [InlineData("cookie: $Path", true)]
+        [InlineData("cookie: $Domain", true)]
+        [InlineData("cookie: $Port", true)]
+        [InlineData("cookie: name=value; domain=.domain.com", true)]
+        [InlineData("cookie: name=value; expires=invaliddate", true)]
+        public async Task Cookies_GetInvalid_ThrowsCookieException(string cookieString, bool addedInNetfx)
+        {
+            await GetRequest("POST", null, new[] { cookieString }, (_, request) =>
+            {
+                if (PlatformDetection.IsFullFramework)
+                {
+                    // Cookies with an empty name aren't added.
+                    if (addedInNetfx)
+                    {
+                        Assert.NotEmpty(request.Cookies);
+                    }
+                    else
+                    {
+                        Assert.Empty(request.Cookies);
+                    }
+                }
+                else
+                {
+                    Assert.Throws<CookieException>(() => request.Cookies);
+                }
```

Comment:
This doesn't look like a good difference in behavior.  Shouldn't there just be an ActiveIssue on the test until it can be fixed?